src/components/select2/index.tsx

```diff
@@ -243,7 +223,11 @@ const Select: FC<SelectProps> = ({
         needSwap={needSwap}
       >
         {isOpen && (
-          <DropdownWrapper refEl={containerRef?.current || undefined}>
+          <DropdownWrapper
+            refEl={containerRef?.current || undefined}
```

Comment:
nit: no need to include `undefined` since you have already add optional chaining